test/test_cuda.py

```diff
@@ -5968,8 +5968,187 @@ def test_cuda_module_loading_env(self):
         self.assertEqual(val, "LAZY")
 
 
+class TestCompileKernel(TestCase):
+    @unittest.skipIf(not TEST_CUDA, "No CUDA")
```

Comment:
Given the file, I assume this doesn't run when there is no cuda already. Other tests in this file don't have this.